behaviors/wysiwyg.js

```diff
@@ -71,19 +93,41 @@ function splitParagraphs(str) {
  * will have their values returned as text models instead of strings
  * @param  {array} strings
  * @param {array} rules chain of responsibility for paste rules
+ * @param {Element} el needs to be cleared if it throws an error
  * @returns {array}
  */
-function matchComponents(strings, rules) {
+function matchComponents(strings, rules, el) {
   return _.filter(_.map(strings, function (str) {
+    let cleanStr, matchedRule, matchedObj, matchedValue;
+
     // remove extraneous opening <p>, <div>, and <br> tags
     // note: some google docs pastes might have `<p><br>`
-    var cleanStr = str.replace(/^\s?<(?:p><br|p|div|br)(?:\s?\/)?>\s?/ig, ''),
-      matchedRule = _.find(rules, function matchRule(rule) {
-        return rule.match.exec(cleanStr);
-      }),
-      matchedObj, matchedValue;
+    cleanStr = str.replace(/^\s?<(?:p><br|p|div|br)(?:.*?)>\s?/ig, '');
```

Comment:
Hmm! Let me take a look at the v1.0 workflow, and see if that still applies (I think it does)